src/app/view/sharing/useSharing.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+
+import {
+  getFilesToUpload,
+  initSharingMode
+} from '/app/domain/sharing/SharingService'
+
+export const useSharingMode = (): {
+  filesToUpload: Record<string, unknown>[]
+} => {
+  useEffect(() => {
```

Comment:
this effect has to be removed, sharingMode is already init elsewhere (globalAppState)

Reply:
- [x] done

Reply:
Maybe the entire hook can be removed? There seems to be no reactive code here nor any event to listen, so maybe `getFilesToUpload()` can be called directly where needed?